sycl/include/CL/sycl/INTEL/esimd/esimd_enum.hpp

```diff
@@ -28,7 +28,7 @@ using uint = unsigned int;
 // Mark a "ESIMD global": accessible from all functions in current translation
 // unit, separate copy per subgroup (work-item), mapped to SPIR-V private
 // storage class.
-#define ESIMD_PRIVATE __attribute__((opencl_private))
+#define ESIMD_PRIVATE __attribute__((opencl_private)) __attribute__((sycl_explicit_simd))
```

Comment:
@kbobrovs, I put a `sycl_explicit_simd` attribute under `ESIMD_PRIVATE` macro. That way we don't need to modify existing code.

Do you think it's OK, or I should create a new macro for it, say `ESIMD_GLOBAL`? With the new macro it would be something like: 
```
ESIMD_GLOBAL ESIMD_PRIVATE ESIMD_REGISTER(17) int x;
```

Reply:
@DenisBakhvalov  - no, I think it should be ESIMD_PRIVATE. We should have two different macros for the same thing. So your approach seems right to me.